mimic/test/test_identity_auth.py

```diff
@@ -684,6 +684,58 @@ def test_rax_kskey_apikeycredentials(self):
         self.assertTrue(
             len(json_body['RAX-KSKEY:apiKeyCredentials']['apiKey']) == 32)
 
+    def test_osksadm_credentials_list(self):
+        """
+        Test OS-KSADM Credentials Listing
+        """
+        core, root = core_and_root([ExampleAPI()])
+        (response, json_body) = self.successResultOf(json_request(
+            self, root, b"GET",
+            "/identity/v2.0/users/1/OS-KSADM/credentials"
+        ))
+        self.assertEqual(response.code, 404)
+        self.assertEqual(
+            json_body['itemNotFound']['message'], 'User 1 not found')
+        creds = {
+            "auth": {
+                "passwordCredentials": {
+                    "username": "HedKandi",
+                    "password": "Ministry Of Sound UK"
```

Comment:
These sample credentials look like an actual company and a concept related to that company.  Could you make this a bit more neutral?